apps/login-register/src/app/login/login.component.html

```diff
@@ -0,0 +1,72 @@
+<section class="login">
+  <div class="login-container">
+    <div class="row">
+      <div class="col-6">
+        <h1 class="title">Accesa Courses</h1>
+        <form class="login-form" [formGroup]="loginForm">
+          <div class="email">
+            <label>Email or UserName</label>
+            <input
+              class="email-input"
+              type="text"
+              formControlName="email"
+              placeholder="Email"
+            />
+            <span
+              *ngIf="(loginForm.controls['email'].touched || loginForm.controls['email'].dirty) && loginForm.controls['email'].errors?.['required']"
```

Comment:
You can move this to the component.ts or into a shared service to improve readability and code

Reply:
done